confd/templates/northbound/log4j2.xml.tmpl

```diff
@@ -14,6 +14,7 @@ Do not change this file, all changes will be lost. Change corresponding template
                  filePattern="{{ getv "/kilda_logging_logfile_path" }}/northbound.log.json.%i.gz">
            <JsonLayout compact="true" eventEol="true" properties="true">
                 <KeyValuePair key="correlation_id" value="${ctx:correlation_id}"/>
+                <KeyValuePair key="timeMillis" value="$${timeMillis:}"/>
```

Comment:
Have you already tried the following solution (similar to https://issues.apache.org/jira/browse/LOG4J2-1752)?
```<KeyValuePair key="timeMillis" value="$${date:UNIX_MILLIS}" />```

Reply:
> Have you already tried the following solution (similar to https://issues.apache.org/jira/browse/LOG4J2-1752)?
> `<KeyValuePair key="timeMillis" value="$${date:UNIX_MILLIS}" />`
I spend 5h to find sollution. About unix_millis we have 
```

Reply:
idk how to get unix timestamp from that https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html